docs/infrastructure/microk8s/install.md

```diff
@@ -0,0 +1,274 @@
+# Installing Codacy on microk8s
+
+## Pre-requisites
+
+* a machine running Ubuntu 18.04 LTS version.
+  * Further versions might work but we recommend using an LTS release.
+* a remote session (ssh) onto your machine of choice.
+
+## Cookbook
+
+Assuming that you are starting from a blank slate, the first step is to install microk8s. Otherwise, please jump on to the [Configuring microk8s](###Configuring-microk8s) section.
+
+### Installing microk8s
+
+1. Ensure that you are working off the root folder: `cd /root`.
+2. Make sure the machine is up to date and also has the `nfs-common` package installed: `apt update && apt install nfs-common -y`.
+3. We must run configure the instance's list of hosts with localhost. This in turn will allow the correct resolution of the local microk8s cluster on the kubeconfig. We can do so by running `sed -i -e "1s/127.0.0.1 localhost.*/127.0.0.1 localhost $(hostname)/" /etc/hosts`.
+4. Traffic forwarding must be configured on the machine so that the local microk8s cluster pods can reach the internet. This can be done by running `iptables -P FORWARD ACCEPT` (more info on this subject can be [found here](https://github.com/ubuntu/microk8s/tree/1.12#my-pods-cant-reach-the-internet-but-my-microk8s-host-machine-can)).
```

Comment:
This does not appear anymore in `master` of the project, can we instead indicate a minimum version supported of microk8s? (looks like the workaround is not needed > 1.14)

Reply:
they seem to have moved the documentation to https://microk8s.io/docs/troubleshooting#common-issues

i am replacing the link as this issue is still documented there.